meson.build

```diff
@@ -228,4 +226,4 @@ install_data(['nvmf-autoconnect/udev-rules/70-nvmf-autoconnect.rules',
               'nvmf-autoconnect/udev-rules/71-nvmf-iopolicy-netapp.rules'],
              install_dir: udevrulesdir)
 install_data(disc,
-             install_dir: join_paths(sysconfdir, 'nvme'))
+             install_dir: join_paths(get_option('sysconfdir'), 'nvme'))
```

Comment:
Don't think this is correct. The path install path needs to be prefixed. This would install it hard under /etc/nvme/discovery.conf. For example if the user uses --prefix=/tmp/ the discovery.conf has to go to /tmp/etc/nvme/discovery.conf.

Reply:
This is the reason I did remove prefix from it.

1. documentation & source file defined without prefix
https://github.com/linux-nvme/nvme-cli/blob/master/fabrics.c#L45
https://github.com/linux-nvme/nvme-cli/blob/master/Documentation/nvme-discover.txt#L43
https://github.com/linux-nvme/nvme-cli/blob/master/Documentation/nvme-connect-all.txt#L41
https://github.com/linux-nvme/nvme-cli/blob/master/nvmf-autoconnect/systemd/nvmf-autoconnect.service#L3

2. Released packages add files on /etc (not /usr/etc)

3. rpm macros does not include prefix on sysconfdir by default.

Reply:
Good points. First thing, The hard coded values are wrong. We need to make the paths configurable. Next thing is to figure out the correct location (see prefix vs destdir discussion, e.g. https://unix.stackexchange.com/questions/444211/what-is-the-diffference-between-the-prefix-and-destdir-params-for-dh-auto-instal). On point 3. let me read up on this one. I wouldn't be surprised if there is a catch to it (e.g. distros are not agreeing on this...)

I'll send patches for point 1.

Reply:
I revert that and add prefix on rpmbuild :)

Reply:
I create issue before your patches committed #1427